contracts/treasury/src/contract.rs

```diff
@@ -43,7 +43,7 @@ pub struct MigrateMsg {}
 pub fn migrate(deps: DepsMut<'_>, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
     versioning::update_software(deps.storage, version!(CONTRACT_STORAGE_VERSION))?;
 
-    Ok(Response::default())
+    Ok(Response::default().set_data(to_binary(sdk::RELEASE_VERSION)?))
```

Comment:
@KirilMihaylov there've been long time since you've done copy/paste in such a scale :-).

Let's replace it with a common function.